content/posts/tau/index.md

```diff
@@ -18,12 +18,92 @@ summary: "A customize osu! ruleset with a circle surrounding a paddle and some n
 - GitHub Repositories : [Altenhh/tau](https://github.com/Altenhh/tau/)
 - Discord : [tau](https://discord.com/invite/7Y8GXAa)
 
-Tau is a custom ruleset with a circle surrounding a paddle and some notes. Based on idea from [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by Laica.
+Tau is a custom ruleset with a circle surrounding a paddle and some notes.
+
+ *The idea is based on [pizzazip's Tau](https://deadlysprinklez.itch.io/tau). The ruleset is developed by [Laica](https://github.com/Altenhh).*
 
 # gameplay
 
 Tau's gameplay revolves around a paddle which is moved around a circle with the mouse. In the circle, there are different types of notes trying to escape it. The player has to move the paddle to block these notes, then hit them using keyboard keys.
-You can see all note types in [Tau gameplay note types wiki](https://github.com/Altenhh/tau/wiki/Note-types).
+
+The gameplay playfield looks like the following:
+
+![Playfield](Images/Playfield.png)
+
+This is an empty playfield, the parts it contains are the following:
+
+![Cursor](Images/Cursor.png) ![Paddle](Images/Paddle.png)
+
+ This is the cursor   /       This is the paddle
```

Comment:
![Screen Shot 2564-05-03 at 03 27 14](https://user-images.githubusercontent.com/68165621/116826682-77168580-abbf-11eb-8fc6-1156b97e87fb.png)
It's look like this. I think put it in seperate line is better.
```suggestion
![Cursor](Images/Cursor.png)
*This is the cursor*
![Paddle](Images/Paddle.png)
*This is the paddle*
```